app/styles/components/_notifications.scss

```diff
@@ -0,0 +1,153 @@
+$notifications-dropdown-content-color: $wds-color-light-c7;
+$wds-icon-color: $wds-color-light-c5;
+
+$content-padding: 18px;
+
+.wds-notifications {
```

Comment:
should this be component dedicated to notifications only? Maybe we should extract a part that can be used in other use-cases?